ocrd_tool.md

```diff
@@ -59,6 +73,16 @@ properties:
           executable:
             description: The name of the CLI executable in $PATH
             type: string
+          input_file_grp:
+            description: Input fileGrp@USE this tool expects by default
+            type: array
+            items:
+              type: string
+              pattern: '^OCR-D-[A-Z0-9-]+$'
+          output_file_grp:
+            description: Output fileGrp@USE this tool produces by default
+            type: string
+            pattern: '^OCR-D-[A-Z0-9-]+$'
```

Comment:
Typo?: [A-Z0-9\\-]
The backslash inside the square bracket has to be escaped by an additional backslash.